backend/services/formatResearchData.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Function for converting research data.
+ * @param {Object} researchData - Research data
+ * @returns {string[]} Formatted research data
+ */
+function formatResearchData (researchData) {
+    let formattedResearchData = '';
+
+    for (const key in researchData[0])
```

Comment:
If `researchData` is `object` as stated in jsdoc, how can it have index 0?

Reply:
fixed docstring